File: JavaScript/0349_intersection_of_two_arrays/solution.js
function intersection_of_two_arrays(nums1,num2) {
    let result = new Set();
    if (nums1.length >= nums2.length) {
        for (let num of nums1) {
            if (num2.includes(num)) {
                result.add(num);
            }
        }
    } else {
        for (let num of nums2) {
            if (nums1.includes(num)) {
                result.add(num);
            }
        }
    }
    result = Array.from(result);
    return result;
}

const nums1 = [4,9,5];
const nums2 = [9,4,9,8,4];
console.log(intersection_of_two_arrays(nums1,nums2));

File: Python/0349_intersection_of_two_arrays/solution.py
def intersection_of_two_arrays(nums1,nums2):
    result = list()
    if len(nums1) >= len(nums2):
        for num in nums1:
            if num in nums2:
                result.append(num)
    else:
        for num in nums2:
            if num in nums1:
                result.append(num)
    result = list(set(result))
    return result
            
nums1 = [4,9,5]
nums2 = [9,4,9,8,4]
print(intersection_of_two_arrays(nums1,nums2))